test/files.js

```diff
@@ -30,13 +35,30 @@ const jsOptions = {
   args: ['--enable-sharding-experiment']
 }
 
+/**
+ * @param {Controller} daemon
+ * @param {string} path
+ * @param {Parameters<Controller["api"]["files"]["mkdir"]>[1]} [options]
```

Comment:
Maybe add a typedef for `Controller["api"]` to reduce some of the typings below?